spec/features/user_adds_a_new_item_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+feature 'User Adds A New Item' do
+  scenario 'Signed in User' do
+    user = FactoryGirl.create(:user)
+    login_as(user, :scope => :user)
+
+    visit new_item_path 
+    fill_in "Name", with: "Rad Item"
+    fill_in "Description", with: "The best thing ever.\n Better than everything"
+
+    click_button "Add Thing(r)"
+    expect(page).to have_content('Thing(r) added')
```

Comment:
internationalization starts with an "i", then has 18 letters and ends with an "n" :)